addon/objects/uml-primitives/fd-uml-template-class.js

```diff
@@ -0,0 +1,139 @@
+/**
+  @module ember-flexberry-designer
+*/
+
+import Ember from 'ember';
+
+import { Class } from './fd-uml-class';
+import FdUmlElement from './fd-uml-element';
+
+/**
+  An object that describes a class on the UML diagram.
+
+  @class FdUmlTemplateClass
+  @extends FdUmlElement
+*/
+export default FdUmlElement.extend({
```

Comment:
You can extend the class `FdUmlClass` and add only the `params` property to it.
There is no support for the `collapsed` state.